MeetupPlatformApi/Migrations/20220401115515_MeetupPKFromIntToGuid.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata;
+
+#nullable disable
+
+namespace MeetupPlatformApi.Migrations
+{
+    public partial class MeetupPKFromIntToGuid : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropPrimaryKey(
+                name: "pk_meetups",
+                table: "meetups");
+
+            migrationBuilder.DropColumn(
+                name: "id",
+                table: "meetups");
+
+            migrationBuilder.AddColumn<Guid>(
+                name: "id", 
+                table: "meetups",
+                nullable: false,
+                defaultValue: Guid.Empty);
```

Comment:
This will fail if we have 2+ meetups in our DB.

Reply:
Fixed.